Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -595,9 +595,10 @@ def is_usable(self, force_update=False):
         if not service:
             _log.debug(u'{} has no service to check if the device is usable'.format(self.device_type.software_variant))
             return True
-
-        exit_code = self.executive.run_command([SimulatedDeviceManager.xcrun, 'simctl', 'spawn', self.udid, 'launchctl', 'list', service], return_exit_code=True)
+        for service in service:
+            exit_code = self.executive.run_command([SimulatedDeviceManager.xcrun, 'simctl', 'launch', self.udid, service], return_exit_code=True)
         if exit_code == 0:
+            self.executive.run_command([SimulatedDeviceManager.xcrun, 'simctl', 'terminate', self.udid, service], return_exit_code=True)
             return True
```

Comment:
If you do have a reason for iterating through multiple services, it seems like this `if` condition needs to be inside the `for` block.

Reply:
Terminating the app that's on the screen will send us back to the Home Screen so the test runner can be installed. This terminates whatever app has been launched; if we wanted to terminate each service we could put it in the for loop; should work either way, but I think dropping the loop and leaving this here would be better.